src/main/scala/net/jcazevedo/moultingyaml/YamlPrinter.scala

```diff
@@ -0,0 +1,67 @@
+package net.jcazevedo.moultingyaml
+
+import org.yaml.snakeyaml.{DumperOptions, Yaml}
+
+/**
+  * @author Simone D'Avico (simonedavico@gmail.com)
```

Comment:
If you want, you should also provide author information for the remaining entities you've created.